excel_import_export_demo/import_sale_orders/templates.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2019 Ecosoft Co., Ltd.
+  License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo>
+    <record id="import_sale_order_xlsx_template" model="xlsx.template">
+        <field name="res_model">sale.order</field>
+        <field name="fname">import_sale_order.xlsx</field>
+        <field name="name">Import Sale Order Template</field>
+        <field name="description">Sample Import Sales Order Tempalte for testing</field>
```

Comment:
Sample Import Sales Order Template for testing

Reply:
Fix